7.redundancy_analysis/notebooks/0.redundancy_analysis.ipynb

```diff
@@ -0,0 +1,1098 @@
+{
```

Comment:
<p>Are the min and max of the redundancy indices valuable info?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>These were used for plotting to keep a square scale when visualizing the results</p>